### DISCUSSION TITLE
Security matters

### DISCUSSION THREAD

**Discussion by duarch** [Sunoo/homebridge-camera-ffmpeg#1180] [Q&A] [answered]
I have a doubt concerning the "user" and "password" in source value. How secure is this? 
Can some explain this? Who can access this information?

**Comment by Sunoo** [comment, marked as answer]
With default permissions, /var/lib/homebridge/config.json is readable by all users on your system. You could conceivably lock that down some, but it would still be readable by any other Homebridge plugin. I’m not aware of any plugin that has ever taken advantage of that though, and this is true of every Homebridge plugin’s config, not just this one.

**Reply by duarch** [reply]
Thank you!
